src/determineConfig.js

```diff
@@ -36,9 +37,14 @@ module.exports = function determineConfig(args, config, getNameForSnapshotUsingT
     update = true;
   }
 
+  if (typeof process !== "undefined" && process.env && process.env.CHAI_JEST_CI) {
```

Comment:
Let's actually just use the environment variable "CI"; a lot of environments set this automatically (Travis for example, if I remember correctly)